vanetza/asn1/fetch_ieee_asn1_files.cmake

```diff
@@ -0,0 +1,116 @@
+if(NOT DIRECTORY)
+    message(FATAL_ERROR "DIRECTORY variable is undefined")
+endif()
+file(MAKE_DIRECTORY "${DIRECTORY}")
+
+file(DOWNLOAD
+    https://forge.etsi.org/rep/ITS/asn1/ieee1609.2/-/raw/2022-published/Ieee1609Dot2.asn
+    "${DIRECTORY}/IEEE1609dot2.asn")
+file(READ "${DIRECTORY}/IEEE1609dot2.asn" _content)
+string(REGEX REPLACE " Extension" " IEEEExtension" _content "${_content}")
```

Comment:
Wow, I-triple-E becomes I-quadruple-E. Maybe go for `IeeeExtension`? Looks more readable in my opinion.